docs/how-to/route-module-type-safety.md

```diff
@@ -45,7 +45,23 @@ If you want to run type checking as its own command — for example, as part of
 }
 ```
 
-## 4. Type-only auto-imports (optional)
+## 4. Typing `AppLoadContext`
+
+
+## Extending app `Context` types
+
+To define your app's `context` type, add the following in a `.td` or `.d.ts` file within your project:
```

Comment:
Should `.td` be `.ts`?

Reply:
```suggestion
To define your app's `context` type, add the following in a `.ts` or `.d.ts` file within your project:
```

Reply:
woops - fixed in https://github.com/remix-run/react-router/pull/12872/files